packager/react-packager/src/DependencyResolver/DependencyGraph/ResolutionRequest.js

```diff
@@ -336,6 +336,8 @@ class ResolutionRequest {
         file = potentialModulePath + '.native.js';
```

Comment:
@vjeux So what will happen once Node.js starts supporting `.mjs` as extension for ES6, as that will mean that the npm eco system will slowly adopt this as well and eventually endup in packages that people might intent to use for React Native. There will be the same conflicts `index.ios.js` vs `index.ios.mjs`